chimera.mk

```diff
@@ -84,17 +84,21 @@ chim-nonfree-init:
 # Include subdir Makefiles
 -include $(CHIM_ROOT)/utils/utils.mk
 # Include target makefiles
+TB_DUT = tb_chimera_soc
 -include $(CHIM_ROOT)/target/sim/sim.mk
 
 #################################
 # Phonies for the entire system #
 #################################
-
-CHIM_ALL += chs-hw-init snitch-hw-init chim-sw chim-bootrom-init chs-sim-all chim-sim
+CHIM_HW_ALL = chs-hw-init snitch-hw-init chim-bootrom-init chs-sim-all
+CHIM_SW_ALL = chim-sw
+CHIM_ALL += $(CHIM_HW_ALL) $(CHIM_SW_ALL) chim-sim
 CHIM_CLEAN += chim-sw-clean chim-sim-clean
 
-.PHONY: chim-all
-chim-all: $(CHIM_ALL)
+.PHONY: chim-all all
+all: chim-all ## Alias for chim-all
+chim-all: $(CHIM_ALL) ## Generate full chimera infrastructure
 
-.PHONY: chim-clean
-chim-clean: $(CHIM_CLEAN)
+.PHONY: chim-clean clean
+clean: chim-clean ## Alias for chim-clean
```

Comment:
What is the purpose of these aliases? I would tend to avoid generic names because we are including many makefiles (all the targets + cheshire, etc..) and we might end up overwriting some other targets we are not aware of.